src/current/releases/index.md

```diff
@@ -374,3 +374,9 @@ During development of a major version of CockroachDB, releases are produced acco
 - Release candidates are produced after the series of beta releases and are nearly identical to what will become the initial generally available (GA) release. Release candidates have `rc` in the version name. Example: `v23.1.0-rc.1`.
 - A major version's initial GA release is produced after the series of release candidates for a major version, and ends with `0`. Example: `v23.1.0`. GA releases are validated and suitable for production environments.
 - Patch (maintenance) releases are produced after a major version's GA release, and are numbered sequentially. Example: `v23.1.13`.
+
+## Licenses
+
+Unless otherwise noted, all binaries available on this page are variously licensed under the Business Source License 1.1 (BSL), the CockroachDB Community License (CCL), and other licenses specified in the source code. To determine whether BSL or CCL applies to a CockroachDB feature, refer to the [Licensing FAQs]({% link {{page.version.version}}/licensing-faqs.md }#feature-licensing) page under Feature Licensing. The default license for any feature that is not listed is the CCL.
```

Comment:
```suggestion
Unless otherwise noted, all binaries available on this page are variously licensed under the Business Source License 1.1 (BSL), the CockroachDB Community License (CCL), and other licenses specified in the source code. To determine whether BSL or CCL applies to a CockroachDB feature, refer to the [Licensing FAQs]({% link {{page.version.version}}/licensing-faqs.md %}#feature-licensing) page under Feature Licensing. The default license for any feature that is not listed is the CCL.
```